assets/overlays/aws-efs/generated/standalone/controller.yaml

```diff
@@ -33,6 +33,7 @@ spec:
     metadata:
       annotations:
         cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes: socket-dir
+        openshift.io/required-scc: privileged
```

Comment:
@RomanBednar the workload is assigned the privileged SCC by default (due to the `privileged: true` in the `securityContext` of the containers), which is why I pinned `openshift.io/required-scc: privileged`. The goal was to avoid changing its existing behavior.

Reply:
@kramaranya I see now, thanks!